packages/table/src/react/__stories__/index.story.js

```diff
@@ -420,3 +414,36 @@ const drawerStory = storiesOf('drawer', module)
       </Drawer>
     </Table>
   ))
+  .add('nested table', _ => {
+    const baseRow = (
+      <Table.Row>
+        <Table.Cell>Parent Cell</Table.Cell>
+        <Table.Cell>Parent Cell</Table.Cell>
+        <Table.Cell>Parent Cell</Table.Cell>
+      </Table.Row>
+    )
+
+    return (
+      <Table>
+        <Drawer base={baseRow} startOpen>
+          <Table inDrawer>
```

Comment:
added a new `inDrawer` prop to add styling. alternative is Drawer could check if child is a table at add prop using cloneElement

Reply:
A few thoughts:

- I want to minimize whatever any component knows about the other.  
- I want to minimize the little "mode" tweaks that devs have to add to props
- I want to minimize total props on components.

In this case, the first point bears the most weight for me in general.  

The Table already knows about Drawer (it horz pads its own rows if it has Drawer children, which have horz padd as well).  

I think the `inDrawer` prop is probably as good as the known alternate solutions.

So, ^ that's just me talking through it. :)